### ISSUE TITLE
Facebook is logged out when reopening Brave

### ISSUE THREAD

**Issue by d3vilguard** [brave/brave-browser#34282]
## Description 
I can log in to facebook, use it normally, When I close the browser and reopen it, I'm logged out of facebook and I have to login again. Every single time.

## Actual result:
Logged out of  facebook when reopening the browser

## Expected result:
Should be logged in to facebook when reopening the browser.

## Reproduces how often: 
Every single time...

## Brave version (brave://version info)
All of them, stable/beta/nightly.

## Version/Channel Information:

- Can you reproduce this issue with the current release? **YES**
- Can you reproduce this issue with the beta channel?  **YES**
- Can you reproduce this issue with the nightly channel?  **YES**

## Other Additional Information:

- Does the issue resolve itself when disabling Brave Shields? **NO**
- Does the issue resolve itself when disabling Brave Rewards?  **NO**
- Is the issue reproducible on the latest version of Chrome? **IDK for Chrome but Vivaldi was fine**
 
## Miscellaneous Information:
Fedora 39 Gnome. Have deleted cache, config, cookies, tried different options for shields, tracking. Nothing.

**Comment by sokhengpory** [comment]
+1

**Comment by julienkosinski** [comment]
Yes... We can actually reproduce this on every OS (I've tested Windows 10, Windows 11 and Fedora so far, but I'm pretty sure that's the same everywhere).
I think Facebook made some work to avoid Brave from being counted as a trusted device. This is pretty annoying, there's no simple fix (User agent spoofing won't work, playing with Brave Shield and fingerprinting detection won't work either)...

**Comment by 8227846265** [comment]
Same issue here.
